searchQuotes.js

```diff
@@ -0,0 +1,12 @@
+"use strict";
+
+const quotes = require("./quotes");
+
+const searchQuotes = term => {
+    const regExp = new RegExp(term, 'i');
+    return quotes.filter(quote => {
+        return quote.match(regExp);
```

Comment:
Please can you use the implicit return syntax (i.e. remove the braces and `return` keyword) from the callback passed to `quotes.filter`? This is used elsewhere in the codebase where possible, so it would be nice to keep things consistent.